android/lib/build.gradle

```diff
@@ -44,4 +44,6 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test:runner:1.3.0'
     androidTestImplementation 'com.google.truth:truth:1.0.1'
+    androidTestImplementation "androidx.fragment:fragment-testing:1.2.5"
+    androidTestImplementation "org.mockito:mockito-android:2.28.2"
```

Comment:
Minor: Want to keep these sorted? Also, for consistency use single quotes like the other deps unless string interpolation is needed.